Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -113,13 +129,14 @@ def create_session(self, hops=0):
         # the settings dictionary
         settings['outgoing_port'] = 0
         settings['num_outgoing_ports'] = 1
+        extensions = list(DEFAULT_LT_EXTENSIONS)
```

Comment:
This is already a list right?

Reply:
The extensions variable can get modified, but we don't want the default to get modified. Thus this creates a copy of the default list that we can modify.

Reply:
I see, this wasn't clear for me so please add a small comment 👍